tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -934,6 +935,8 @@ def setUp(self):
         self.session_error_metric = indexer.record(org_id, SessionMetricKey.SESSION_ERROR.value)
         self.session_status_tag = indexer.record(org_id, "session.status")
         self.release_tag = indexer.record(self.organization.id, "release")
+        self.tx_metric = indexer.record(org_id, TransactionMetricsKey.TRANSACTION.value)
```

Comment:
As @jjbayer mentioned, there is no `transaction` metric, there is only a `transaction.duration`, and so this test passes because you populated the dataset `transaction` metric